test/sequential/bootstrap.updatePromotion.test.ts

```diff
@@ -82,7 +77,7 @@ beforeAll(async () => {
   await setupBootstrapTokensBalance(bootstrapCurrency, sudo, [testUser1]);
 });
 
-test("bootstrap - bootstrap - Check if we can change promoteBootstrapPool in each phase", async () => {
+test("bootstrap - Check if we can change promoteBootstrapPool in each phase", async () => {
   let checkingUpdatingPool: MangataGenericEvent[];
```

Comment:
plz rename , and: why are you using this global variable ?

Reply:
I need to use this action several times per test.